main.go

```diff
@@ -34,16 +40,18 @@ func main() {
 
 	start := time.Now()
 	config := struct {
-		prefix  string
-		timeout time.Duration
-		public  string
-		output  string
+		prefix     string
+		timeout    time.Duration
+		public     string
+		output     string
+		keysAmount uint64
 	}{}
 
 	flag.StringVar(&config.prefix, "prefix", "AY/", "prefix of base64-encoded public key")
 	flag.DurationVar(&config.timeout, "timeout", 0, "stop after specified timeout")
 	flag.StringVar(&config.public, "public", "", "start from specified public key")
 	flag.StringVar(&config.output, "output", "", "use \"offset\" to print offset only")
+	flag.Uint64Var(&config.keysAmount, "keys", 1, "amount of keys that will be returned. 0 means infinite")
```

Comment:
This could be achieved by running the tool in the loop
```bash
for i in {1..100} ; do wireguard-vanity-key --prefix=foo ; done
```

Reply:
That's horrible

Reply:
This tool spends minutes/hours/days crunching candidate keys so 1 microsecond to start a process is nothing.